src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const rightRobot = {};
```

Comment:
```suggestion
  const inversedRobot = {};
```

Reply:
I can't change variable on this name, cause fall linter in VSC.
 I will leave the old one.